Doc/library/struct.rst

```diff
@@ -292,24 +302,40 @@ Notes:
    format <half precision format_>`_ for more information.
 
 (7)
-   For padding, ``x`` inserts null bytes.
-
+   For padding, ``'x'`` inserts one NUL byte.
```

Comment:
Slightly OT: I've often suspected that "For padding" here was a typo, and the intended meaning was "For packing" (since for unpacking, of course, nothing is being inserted).